glossary.html

```diff
@@ -61,13 +61,11 @@ <h1>Acronym Glossary</h1>
                 <p id="page-copyright">
                     &copy; 2019 <a href="mailto:nathan.foglia2@student.saultcollege.ca?subject=Messgae%20from%20CSD120%20Website"> Nathan Foglia</a>
                 </p>
-                <picture>
-                <source media= "(min-width: 1020px)" srcset = "images/sc-logo@4x.png">
-                <source media= "(min-width: 835px)" srcset = "images/sc-logo@3x.png">
-                <source media= "(min-width: 650px)" srcset = "images/sc-logo@2x.png">
-                <source media= "(min-width: 465px)" srcset = "images/sc-logo@1.5x.png">
-                <img src="images/sc-logo.png" alt="Sault College Logo Image" style ="width:auto">
-           </picture>
+                <img class="sc-logo" 
+                    src="images/sc-logo.png" 
+                    sizes="70px" 
+                    srcset="images/sc-logo.png 70w, images/sc-logo@1.5x.png 105w, images/sc-logo@2x.png 140w, images/sc-logo@3x.png 210w, images/sc-logo@4x.png 280w" 
+                    alt="Sault College Logo">
```

Comment:
Using the `<picture>` element as you have it doesn't give you the automatic display density adaptation like the sizes/srcset attributes do.